### PR TITLE
AC-1503 Application Details View Update

### PR DESCRIPTION
**Pull request by lillie-dae**
### Description

Fixes # .

A few sentences describing the changes proposed in this pull request.

### Status
**Ready/Work in progress/Hold**

### Types of changes

- [ ] Non-breaking change (fix or new feature that would not break existing functionality).
- [ ] Breaking change (fix or new feature that would cause existing functionality to change).
- [ ] New unit tests added to cover the changes.
- [ ] New e2e tests added to cover the changes. (If it has been decided these will not be added with the PR, a separate ticket **must** be created and linked in the ticket below before merge)
- [ ] All tests passed locally.

### CODE REVIEW

File: src/components/AppRepo/DetailView/DetailView.vue [hunk 1]
```diff
@@ -20,30 +20,47 @@
...
         <v-container
             style="max-width: 100%; height: calc(100vh - 210px); overflow: auto"
             data-cy="detail-sub-module"
         >
             <v-container style="max-width: 75%">
-                <DetailHeader :versionDetails="version_details"></DetailHeader>
+                <!-- PICTURE -->
+                <!-- HEADER 1 -->
+                <!-- TAGS -->
+                <!-- HEADER 2 -->|
```
Comment (rowellx68) [inline src/components/AppRepo/DetailView/DetailView.vue:32]: Do we still need these?

**Review by rowellx68 — COMMENTED** [review]
Having had a look ad @DanielNuttall-AD's PR, the endpoints should be `/app-store/*`

https://github.com/AI4VBH/AIDE-api/blob/16440dd00f290726054cc2a45749f037e8bcec35/src/modules/store/applications/applications.controller.ts#L29

File: src/components/AppRepo/DetailView/DetailView.vue [hunk 2]
```diff
@@ -86,39 +103,32 @@ import { Application, Version, VersionDetails } from "@/models/AppRepo/Applicati
...
-            .catch((err) => {
-                console.log(err);
-                this.loading = false;
-            });
+        try {
+            const application = await getApplicationDetails(application_id, version_id);
+            this.loading = false;
+            this.application = application;
+        } catch (error) {
+            console.log(error);
```
Comment (mehranrashid) [inline src/components/AppRepo/DetailView/DetailView.vue:129]: Do we need this still

Comment (lillie-dae) [reply src/components/AppRepo/DetailView/DetailView.vue]: No I've removed that.

**Review by rowellx68 — APPROVED** [review]

**Pull request merged**